crates/settings/src/settings.rs

```diff
@@ -35,9 +35,12 @@ pub fn default_settings() -> Cow<'static, str> {
 #[cfg(target_os = "macos")]
 pub const DEFAULT_KEYMAP_PATH: &str = "keymaps/default-macos.json";
 
-#[cfg(not(target_os = "macos"))]
+#[cfg(all(not(target_os = "macos"), not(target_os = "windows")))]
```

Comment:
This should probably just be used on Linux:

```suggestion
#[cfg(target_os = "linux")]
```